Assets/VRTK/Scripts/Utilities/ObjectFollow/VRTK_TransformFollow.cs

```diff
@@ -155,11 +161,20 @@ protected virtual void CacheTransforms()
             if (gameObjectToFollow == null || gameObjectToChange == null
                 || (transformToFollow != null && transformToChange != null))
             {
-                return;
+                if(positionToFollow == Vector3.zero)
```

Comment:
Code convention: Change to `if( positionToFollow == Vector3.zero)`. Note the space before the parentheses.